frontend/src/styles/MPbuttons.css

```diff
@@ -0,0 +1,31 @@
+.buttonGroup {
+    width: 100%;
+    text-align: center;
+}
+.mp-button{
+    display: inline-block;
+    border-radius: 8px;
+    width: 6em;
+    height: 2em;
+    margin: 10px 20px 10px 20px;
+    text-align: center;
+    font-size: x-large;
+    color: white;
+    font-family: var(--primaryFont);
+    font-weight: bold;
+    border-color: var(--primaryButtonColor);
+}
+#green{
+    background-color: var(--pastelGreen);
+}
+#green:hover{
+    background-color: var(--pastelLightGreen);
+    color: var(--primaryButtonColor);
+}
+#red{
+    background-color: var(--pastelRed);
+}
```

Comment:
just to keep our css files matching, add a space between css items {}.

Reply:
done!